### ISSUE TITLE
Posit to uint64_t conversion 

### ISSUE THREAD

**Issue by MJChku** [stillwater-sc/universal#267]
Is it possible to convert posit to uint64_t which result the same as   *(uint64_t*) &( posit to double) without converting to double first? In fact, I can't use xmm register at conversion.

**Comment by Ravenwater** [comment]
Yes, I can add that to the library. I need to think through the rounding behavior.

**Comment by MJChku** [comment]
Thanks, I have a highly hacked use case of posit number, so I asked.

**Comment by Ravenwater** [comment]
Picked this back up, but now I realize that I do not quite understand the ask. I thought initially that you simply wanted the posit bits returned in a uint64_t.     But (uint64_t) &(posit to double) would imply that you want the address of a double conversion result, which would require a conversion to double, conflicting with your request.

So, I need a little bit more information about what you need in the API.
